pyproject.toml

```diff
@@ -18,9 +18,13 @@ build-backend = "poetry.core.masonry.api"
 python = "^3.8"
 textual = ">=0.41.0"
 pyarrow = ">=7.0.0"
+polars = { version = ">=0.20.0", optional = true } 
```

Comment:
You'll also need to add `polars` as a test dependency in the `tool.poetry.group.test.dependencies` below.

Reply:
(and also as an `additional_dependency` for the `mypy` hook in `pre-commit-config.yaml`)